classy/evaluation/base.py

```diff
@@ -6,11 +6,6 @@
 class Evaluation:
     def __call__(
         self,
-        predicted_samples: List[
-            Tuple[
-                Union[SentencePairSample, SequenceSample, TokensSample, QASample, GenerationSample],
-                Union[str, List[str], Tuple[int, int]],
-            ]
-        ],
+        predicted_samples: List[Union[SentencePairSample, SequenceSample, TokensSample, QASample, GenerationSample]],
```

Comment:
I'm not sure if we should create a grouping typing variable ClassificationSample that is the union of SentencePairSample, SequenceSample, TokensSample, QASample :thinking:

Reply:
Actually, why aren't we using ClassyStruct?

Reply:
if we only use ClassyStruct to subclass samples, I'd use it and rename it to ClassySample maybe

Reply:
I agree with that, we should use ClassySammple in imo. I am changing it